packages/schematics/update/update/npm.ts

```diff
@@ -55,25 +56,48 @@ function readOptions(
     logger.info(`Locating potential ${baseFilename} files:`);
   }
 
-  let options: { [key: string]: string } = {};
+  const options: PackageManagerOptions = {};
   for (const location of [...defaultConfigLocations, ...projectConfigLocations]) {
     if (existsSync(location)) {
       if (showPotentials) {
         logger.info(`Trying '${location}'...found.`);
       }
 
       const data = readFileSync(location, 'utf8');
-      options = {
-        ...options,
-        ...(yarn ? lockfile.parse(data) : ini.parse(data)),
-      };
-
-      if (options.cafile) {
-        const cafile = path.resolve(path.dirname(location), options.cafile);
-        delete options.cafile;
-        try {
-          options.ca = readFileSync(cafile, 'utf8').replace(/\r?\n/, '\\n');
-        } catch { }
+      // Normalize RC options that are needed by 'npm-registry-fetch'.
+      // See: https://github.com/npm/npm-registry-fetch/blob/ebddbe78a5f67118c1f7af2e02c8a22bcaf9e850/index.js#L99-L126
+      const rcConfig: PackageManagerOptions = yarn ? lockfile.parse(data) : ini.parse(data);
+      for (const [key, value] of Object.entries(rcConfig)) {
```

Comment:
nit: Should probably reuse the code. Could you please add a todo?

Reply:
Probably we should remove this package altogether!